packages/lab/src/overlay/OverlayPanel.tsx

```diff
@@ -9,7 +9,7 @@ import { clsx } from "clsx";
 import { useOverlayContext } from "./OverlayContext";
 import { OverlayPanelBase } from "./OverlayPanelBase";
 
-const withBaseName = makePrefixer("saltOverlayPanel");
+const withBaseName = makePrefixer("saltOverlay");
```

Comment:
```suggestion
const withBaseName = makePrefixer("saltOverlayPanel");
```

This should still probably be this

Reply:
This wasn't resolved?